troi/patches/periodic_jams.py

```diff
@@ -14,24 +14,18 @@
 BATCH_SIZE_RECS = 100  # the number of recommendations fetched in 1 go
 MAX_RECS_LIMIT = 1000  # the maximum of recommendations available in LB
 
-WEEKLY_JAMS_DESCRIPTION = """<p>The ListenBrainz Weekly Jams playlist aims to be a playlist of tracks that we believe
-    that you'll like. Designed to be a review playlist, it suits well for playing whenever you need to have comfortable
-    music to listen to that does not require active listening.</p>
+WEEKLY_JAMS_DESCRIPTION = """<p>The ListenBrainz Weekly Jams playlist features songs that you have listened to before, 
+    arranged into a comfortable playlist that doesn't require active listening on your part. 
+    Relax (or party) to familiar favorites.</p>
```

Comment:
I think the "Relax (or party) to familiar favorites." is odd -- we can just remove it and its fine.